### DISCUSSION TITLE
DeepL Support

### DISCUSSION THREAD

**Discussion by kleineluka** [ThijmenDam/BarTranslate#17] [Ideas]
Moving the [open issue](https://github.com/ThijmenDam/BarTranslate/issues/13) to the new discussions page, and also potentially breathing life back into the idea. DeepL is a great tool and would love to see it in this handy little app 🎉

**Comment by ThijmenDam** [comment]
Hi @lukasong, glad to hear the app is useful to you. Comments like these keep me motivated to continue development despite my busy schedule. As for DeepL, I definitely agree that this would be a great addition to the app. I have a little more time on hands next week, so I'll see what I can do.

**Comment by ThijmenDam** [comment]
Hi @lukasong, support for DeepL is now included in the latest release. See #19!
